internal/tsc_wrapped/compiler_host.ts

```diff
@@ -335,6 +336,45 @@ export class CompilerHost implements ts.CompilerHost, tsickle.TsickleHost {
       }
     }
 
+    // The following handles the case when the filename is outside of
+    // bazelOpts.package but still needs to be mapped to the correct
```

Comment:
it seems like the logic above that uses `bazelOpts.package` was too specific. Can this general logic replace some of that?